tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/tasks/SDKGenerationTaskBase.ts

```diff
@@ -0,0 +1,10 @@
+import { DockerTaskEngineContext } from '../DockerTaskEngineContext';
+
+export type TaskType = 'InitTask' | 'GenerateAndBuildTask' | 'MockTestTask';
+
+export interface SDKGenerationTaskBase {
+    taskType: TaskType;
+    context: DockerTaskEngineContext;
+    order: number;
```

Comment:
I am still not convinced that we need the order here. Can it be determined by the order from the config file?

Reply:
If we want to add the order in config file, we need to change the existing config file now. So I think it's better to add the order in config file firstly, and then remove the order here.